apps/workflow-cf.yml.j2

```
@@ -60,6 +66,8 @@ Resources:
             Value: "{{ task['vcpu'] }}"
           - Type: MEMORY
             Value: "{{ task['memory'] }}"
+          - Type: GPU
+            Value: "{{ task['gpu'] if 'gpu' in task else 0 }}"
```

Probably better to just not set this value at all if the task doesn't require it.
```suggestion
          {% if 'gpu' in task %}
          - Type: GPU
            Value: "{{ task['gpu'] }}"
          {% endif %}
```